qemu/tests/cfg/pvpanic_basic.cfg

```diff
@@ -47,7 +47,7 @@
             only Windows
             crash_method = usb_keyboard
             usb_devices += " kb1"
-            usb_type_kb1 = "usb-kbd"
+            usbdev_type_kb1 = "usb-kbd"
```

Comment:
@leidwang Could you help review this case ? Thanks!

Reply:
LGTM.

JOB LOG    : /root/avocado/job-results/job-2022-03-11T04.53-17bc0d8/job.log
 (1/2) Host_RHEL.m.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
| (1/2) Host_RHEL.m.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1164.08 s)
 (2/2) Host_RHEL.m.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win.x86_64.io-github-autotest-qemu.pvpanic_basic.usb_keyboard.q35: STARTED
\ (2/2) Host_RHEL.m.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win.x86_64.io-github-autotest-qemu.pvpanic_basic.usb_keyboard.q35: PASS (206.90 s)